examples/using-javascript-transforms/src/articles/2017-03-09-choropleth-on-d3v4/index.js

```diff
@@ -47,6 +47,7 @@ class choroplethBase extends React.Component {
   }
 
   render() {
+    console.log(this)
```

Comment:
why `console.log` here

Reply:
Using them to test, and forgot to delete them.